microservices/notification/src/config/start.ts

```diff
@@ -10,6 +12,16 @@ const startConfig = GetMsStartConfig(CONST, {
   type: 'microservice',
   registerMethods,
   registerJobs,
+  hooks: {
```

Comment:
look at this to realize how to work with firebase:
https://github.com/Lomray-Software/microservices/blob/staging/microservices/users/src/constants/index.ts#L14

https://github.com/Lomray-Software/microservices/blob/staging/microservices/users/src/services/identity-provider/firebase.ts

Reply:
will this work for projects without firebase configured?

Reply:
yes